### ISSUE TITLE
Bold subject titles in Features and Traits?

### ISSUE THREAD

**Issue by michaelplews** [canismarko/dungeon-sheets#3]
It would be nice to be able to scan the Features & Traits for the titles for quick referencing

**Comment by canismarko** [comment]
Agreed. I looked in to this, and it sounds like you need to set the ``rich text`` flag on the fields in the PDF, which is not enabled in the fillable PDF provided by WotC. I'm hoping to find a way to directly fill the fields in python without having to rely on pdftk as an external dependency, which would take care of issue https://github.com/canismarko/dungeon-sheets/issues/2, and so maybe this feature can be part of it. That will take some thought, though.
